pkg/rebuild/rebuild/rebuildremote.go

```diff
@@ -441,10 +441,13 @@ func doCloudBuild(ctx context.Context, client gcb.Client, build *cloudbuild.Buil
 	bi.BuildID = build.Id
 	bi.Steps = build.Steps
 	bi.BuildImages = make(map[string]string)
-	for i, s := range bi.Steps {
-		bi.BuildImages[s.Name] = build.Results.BuildStepImages[i]
+	buildErr := gcb.ToError(build)
+	if buildErr == nil {
+		for i, s := range bi.Steps {
```

Comment:
do you want to just add a comment here explaining which conditions cause BuildStepImages to be empty?